scripts/build_ngx_mruby.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+NGX_MRUBY_VERSION='2.2.3'
+NGX_MRUBY_URL="https://github.com/matsumotory/ngx_mruby/archive/v${NGX_MRUBY_VERSION}.tar.gz"
+
+echo "Building ngx_mruby v${NGX_MRUBY_VERSION} for ${STACK}"
+
+BUILD_DIR=$(mktemp -d /tmp/ngx_mruby.XXXX)
+SCRIPTS_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+OUTPUT_DIR="$(dirname "${SCRIPTS_DIR}")/archives/${STACK}"
+
+mkdir -p "${OUTPUT_DIR}"
+cd "${BUILD_DIR}"
+
+echo "Downloading ngx_mruby from ${NGX_MRUBY_URL}"
+curl -sSfL "${NGX_MRUBY_URL}" | tar -xz --strip-components 1
+
+./build.sh
+make install
+
+echo 'nginx build complete!'
+
+NGINX_BIN_DIR="${BUILD_DIR}/build/nginx/sbin"
+cd "${NGINX_BIN_DIR}"
+
+# Check that nginx can start
+./nginx -v
+
+NGINX_VERSION=$(./nginx -v |& cut -d '/' -f 2-)
```

Comment:
Does this mean we don't control version of nginx anymore?

Reply:
@hone Yeah that's correct - ngx_mruby has a default nginx version that they bump periodically (there's a newer version on their `master` for example). At least until there is a time where we need a specific nginx it seemed simpler to use the default given that's the version they test against.

Reply:
I'm pretty sure I was building nginx independent of it so we can update nginx in theory outside of being tied to ngx_mruby.